src/doc.rs

```diff
@@ -0,0 +1,354 @@
+use super::ast::{Arg, ArgNames, Statement, TypeAst, TypedModule};
```

Comment:
This could be a directory

Reply:
What do you mean?

Reply:
Sorry, this file is large, and could be split into smaller files

Reply:
No need to be sorry, this is great :)